core/kotlinx-coroutines-core/src/actors/TypedActor.kt

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2016-2018 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.coroutines.experimental.actors
+
+import kotlinx.coroutines.experimental.*
+import kotlinx.coroutines.experimental.channels.*
+import kotlin.coroutines.experimental.*
+
+/**
+ * [TypedActor] is the base for all stateful actors, which can process only one [type][T] of messages.
+ * [TypedActor] has well-defined lifecycle described in [ActorTraits].
+ * [TypedActor.receive] method is used to declare a message handler, which is parametrized by [T]
+ * to provide better compile-type safety.
+ *
+ * Example:
+ * ```
+ * class ExampleActor : TypedActor<String>() {
+ *
+ *   override suspend fun receive(string: String) = act {
```

Comment:
>> Except I still need to use act right?

> No, TypedActor doesn't have that method

Then I guess this example is wrong and `act` is not needed here.

Reply:
yup, documentation leftover